src/Bridge/Doctrine/Common/Filter/SearchFilterTrait.php

```diff
@@ -136,7 +136,7 @@ protected function getIdFromValue(string $value)
     protected function normalizeValues(array $values, string $property): ?array
     {
         foreach ($values as $key => $value) {
-            if (!\is_int($key) || !\is_string($value)) {
+            if (!\is_int($key) || \is_array($value)) {
```

Comment:
What value will not fit the first part of the IF then ? I mean an array will already be validated by the first part of the OR condition isn't it ?

Reply:
First part of the IF it's the $key, second part it's about the $value. If $value is an INT then a notice will appear. I wrote a test about that here: 

https://github.com/api-platform/core/pull/4295/commits/d33159fc1af1c9ac3165a811ffa30c8f8d7cdf7c